languages.toml

```
@@ -2117,3 +2117,23 @@ roots = []
 comment-token = "#"
 indent = { tab-width = 4, unit = "    " }
 grammar = "python"
+
+[[language]]
+name = "MSBuild"
+scope = "source.xml"
+injection-regex = "xml"
+file-types = [".proj", ".vbproj", ".csproj", ".fsproj", ".targets", ".props"]
```

```suggestion
file-types = ["proj", "vbproj", "csproj", "fsproj", "targets", "props"]
```

These are parsed as file extensions so they don't need the leading `.`

---

Awesome, fixed, thanks.